lib/neo4j/active_node/has_n/association.rb

```diff
@@ -17,6 +17,14 @@ def initialize(type, direction, name, options = {type: nil})
           apply_vars_from_options(options)
         end
 
+        def derive_model_class
+          return @model_class unless @model_class.nil?
+
+          return nil if relationship_class.nil?
+          return false if relationship_class.to_class.to_s.to_sym == :any
```

Comment:
Should this always be `to_class`? Should the direction change based on the current association's direction?